src/main/scala/stage/phases/TransformAnnotations.scala

```diff
@@ -17,6 +17,9 @@ class TransformAnnotations extends Phase with PreservesAll[Phase] with HasRocket
 
   override def transform(annotations: AnnotationSeq): AnnotationSeq = {
     /** Construct output file annotation for emission */
-    new ChiselOutputFileAnnotation(view[RocketChipOptions](annotations).longName.get) +: annotations
+    new ChiselOutputFileAnnotation(
+      view[RocketChipOptions](annotations).longName.get +
+      view[RocketChipOptions](annotations).chiselFileExtension.getOrElse("")
```

Comment:
```suggestion
      view[RocketChipOptions](annotations).chiselFileExtension
```

Reply:
Need to remove this too if we don't have it as an Option.

Reply:
Yep, missed that.